app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -49,23 +48,22 @@ static Dimelo setupDimelo(Context context) {
             dimelo.setUserIdentifier(userIdVal);
         }
 
-        if (isHmsAvailable(context)) {
-            dimelo.setPushNotificationService("hms");
-        } else {
-            FirebaseApp.initializeApp(context);
-            FirebaseMessaging.getInstance().getToken()
-                    .addOnCompleteListener(new OnCompleteListener<String>() {
-                        @Override
-                        public void onComplete(Task<String> task) {
-                            if (task.isSuccessful()) {
-                                String refreshedToken =  task.getResult();
-                                if (refreshedToken != null) {
-                                    Dimelo.getInstance().setDeviceToken(refreshedToken);
-                                }
-                            }
-                        }
-                    });
-        }
+        // if (isHmsAvailable(context)) {
+        //     dimelo.setPushNotificationService("hms");
+        // } else {
+        //     FirebaseApp.initializeApp(context);
+        //     FirebaseMessaging.getInstance().getToken().addOnCompleteListener(new OnCompleteListener<String>() {
+        //         @Override
+        //         public void onComplete(Task<String> task) {
+        //             if (task.isSuccessful()) {
+        //                 String refreshedToken =  task.getResult();
+        //                 if (refreshedToken != null) {
+        //                     Dimelo.getInstance().setDeviceToken(refreshedToken);
+        //                 }
+        //             }
+        //         }
+        //     });
+        // }
```

Comment:
It's faster like that to remove the comment if necessary

```suggestion
        /* if (isHmsAvailable(context)) {
            dimelo.setPushNotificationService("hms");
        } else {
            FirebaseApp.initializeApp(context);
            FirebaseMessaging.getInstance().getToken().addOnCompleteListener(new OnCompleteListener<String>() {
                @Override
                public void onComplete(Task<String> task) {
                    if (task.isSuccessful()) {
                        String refreshedToken =  task.getResult();
                        if (refreshedToken != null) {
                            Dimelo.getInstance().setDeviceToken(refreshedToken);
                        }
                    }
                }
            });
        } */
```

Reply:
you're right it's easier to remove as a block comment, done in [3e14c52](https://github.com/ringcentral-tutorials/engage-digital-messaging-android-demo/pull/27/commits/3e14c5238fabf91ecc3a5ed1d5116c75c88d5c7b)